### PR TITLE
Clarify explanation of how the addon works

### PR DESCRIPTION
**Pull request by 0xadada**
Just some copy cleanup

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -33,7 +33,7 @@ or if you installed as a command run
 
 `ember update`
 
-This will update your app or addon to the latest Ember CLI version. It does this by grabbing your current installed Ember CLI version and looking up the latest version, then applying a diff of the changes needed to your project. Your project files will only get modified if they were changed in the by Ember CLI between versions, and it will only change the section necessary, not the entire file.
+This will update your app or addon to the latest Ember CLI version. It does this by fetching the latest version and comparing it to your projects Ember CLI version. It then applies a diff of the changes from the latest verion to your project. It will only modify the files if there are changes between your projects' version and the latest version, and it will only change the section necessary, not the entire file.
```
Comment (kellyselden) [inline README.md]: projects -> project's

Comment (kellyselden) [reply README.md]: verion -> version

Comment (kellyselden) [reply README.md]: projects' -> project's

**Comment by 0xadada** [comment]
changes added, committed & rebased

**Comment by kellyselden** [comment]
Thanks a bunch!

**Comment by 0xadada** [comment]
Thank @kellyselden for the useful addon.

_tips fedora_

**Pull request merged**